agora_analytics/README.md

```diff
@@ -0,0 +1,41 @@
+# Agora Analytics
+
+## Overview
+
+Agora Analytics tracks and interprets real-time voice and video usage, quality, and performance. Analytics is an extension for Agora’s Voice Calling, Video Calling, and Interactive Live Streaming that helps to locate quality issues, find root causes, and fix issues to improve the end-user experience. 
+    
+When you enable the Analytics integration in Datadog and Agora Console, Analytics pushes usage, quality, and performance data directly to your Datadog account.
+
+## Setup
+
+### Installation
+
+The Agora Analytics integration emits metrics to Datadog. No installation is required on the Datadog side.
```

Comment:
I'd recommend removing this and leaving the configuration section below, as these two have conflicting info.
```suggestion
```

Reply:
removed